backend/src/libs/packages/openai/openai.package.ts

```diff
@@ -0,0 +1,48 @@
+import { OpenAI as OpenAIClient } from 'openai';
+
+import { type IConfig } from '../config/config.js';
+import { type CompletionConfig } from './libs/types/completion-config.type.js';
+
+class OpenAIService {
+  private config: IConfig;
+  private openAIClient: OpenAIClient;
+
+  private DEFAULT_COMPLETION_MODEL = 'gpt-3.5-turbo';
```

Comment:
Let's move model to env variables - we may swap between them to check the resulting prompts

Reply:
Yep, let's specify default but let's specify instead of `gpt-3.5-turbo` some lower model like `babbage` or `ada`

Reply:
@andanf-e I just checked the docs and it says that such models like Babbage and Ada are legacy now and have a little bit different API to make a prompt comparing to GPT-3.5+ models. So, should I create some flag env variable like USE_LEGACY_MODEL, and use the corresponding API based on the value of that flag then?
![image](https://github.com/BinaryStudioAcademy/bsa-2023-writorium/assets/71723893/969fe242-362c-4ec1-a2cd-e9171b3e323d)

Reply:
Sorry, if so, then lets keep it as it is 🥲